packages/@react-aria/selection/src/ListKeyboardDelegate.ts

```diff
@@ -66,7 +66,7 @@ export class ListKeyboardDelegate<T> implements KeyboardDelegate {
     key = this.collection.getKeyAfter(key);
     while (key != null) {
       let item = this.collection.getItem(key);
-      if (item.type === 'item' && !this.disabledKeys.has(key)) {
+      if (item.type === 'item' && !this.disabledKeys.has(key) && !item?.props?.isDisabled) {
```

Comment:
Should this take `disabledBehavior` into account? If `disabledBehavior` is only `selection` then keyboard navigation should still focus disabled items but since this checks the row's prop directly it will skip that row. Guess we'd need to pass selection manager to the delegates now then...

It feels to me that SelectionManager's `isDisabled` should be the source of truth, ideally we'd replace places where `disabledKeys` gets returned as a pass through of `props.disabledKeys` with SelectionManager's isDisabled but that feels potentially breaking?

Reply:
Looks like that's handled in `useSelectableList`, which passes in an empty set if `disabledBehavior` is `selection`: https://github.com/adobe/react-spectrum/blob/1293adf00a11ecac84ec9ab19af7dcea970af2b2/packages/%40react-aria/selection/src/useSelectableList.ts#L58